### ISSUE TITLE
Relative url()s to files within src/ do not work

### ISSUE THREAD

**Issue by phillipskevin** [donejs/css#59]
If you create a `url()` from an image that is within your project's `src/` folder like:
```
background-image: url("./assets/1-banner.jpg");
```

in DoneJS 0.9, the CSS that is injected into the SSR'd page would have:

```
background-image: url(/src/assets/1-banner.jpg);
```

in DoneJS 1.0, the SSR'd page has
```
background-image: url("assets/1-banner.jpg");
```

which is not correct.

**Comment by phillipskevin** [comment]
In done-css 2, this was handled here: https://github.com/donejs/css/blob/v2.0.3/css.js#L107-L109.

**Comment by phillipskevin** [comment]
done-css is using steal-css now and this code is skipped during SSR: https://github.com/stealjs/steal-css/blob/master/css.js#L157-L161.

**Comment by phillipskevin** [comment]
This is fixed in steal-css@1.2.2.
